packages/core/src/App/Containers/RealAccountSignup/account-wizard.jsx

```diff
@@ -314,6 +313,9 @@ const AccountWizard = observer(props => {
             delete clone.tax_identification_number;
         }
 
+        if (clone?.tnc_acceptance === true) {
```

Comment:
```suggestion
        if (clone?.tnc_acceptance) {
```
will it work?

Reply:
Honestly, I did that first and it sent `tnc_acceptance:true` funny, how its behaving. But I'll change it 🙏 since, I tested it again and its going fine 😭